### PR TITLE
add Quantizable Transformers

### PR DESCRIPTION
**Pull request by zhimin-z**

### CODE REVIEW

**Review by HuangOwen — CHANGES_REQUESTED** [review]
Thanks for the commits, some suggestions before merging:
1. Please put this paper in the correct position (publication/arxiv sequence in time)
2. Please use the arxiv page, not the pdf page as the link to [Paper]
3. This paper has a code repo (https://github.com/Qualcomm-AI-research/outlier-free-transformers) although it's empty, we can add this repo as [Code]

**Comment by zhimin-z** [comment]
> Thanks for the commits, some suggestions before merging:
> 
> 1. Please put this paper in the correct position (publication/arxiv sequence in time)
> 2. Please use the arxiv page, not the pdf page as the link to [Paper]
> 3. This paper has a code repo (https://github.com/Qualcomm-AI-research/outlier-free-transformers) although it's empty, we can add this repo as [Code]

image

File: README.md
Comment (HuangOwen) [inline README.md:1]: Again, please put this paper in the correct position (publication/arxiv sequence in time). Do not always put a paper at the beginning of a section. In this case, this NeurIPS 2023 paper should appear before all EMNLP papers and the NeurIPS workshop papers. 

More concretely, this paper should appear before "Efficient LLM Inference on CPUs" and after "Dissecting the Runtime Performance of the Training, Fine-tuning, and Inference of Large Language Models", which is the correct order of their publication date.

**Review by HuangOwen — CHANGES_REQUESTED** [review]
Again, please put this paper in the correct position (publication/arxiv sequence in time). Do not always put a paper at the beginning of a section. In this case, this NeurIPS 2023 paper should appear before all EMNLP papers and the NeurIPS workshop papers.

More concretely, this paper should appear before "Efficient LLM Inference on CPUs" and after "Dissecting the Runtime Performance of the Training, Fine-tuning, and Inference of Large Language Models", which is the correct order of their publication date.

**Review by HuangOwen — APPROVED** [review]

**Pull request merged**